opossum-tool/OpossumUI
# feat: offer package name suggestions

### Summary of changes

We add the suggestions from deps.dev to the autocomplete for the package name field.

### Context and reason for change

#2394 

### How can the changes be tested

Create a new attribution; type "react" for the package name and check that there are nine suggestions from deps.dev.
